### PR TITLE
feat : added authentication component

### PR DESCRIPTION
**Pull request by Priya-Patel31**
It consists of :

* Signup
* Signin
* Forgot Password

### CODE REVIEW

**Review by t007rushi — APPROVED** [review]
Go ahead merge it Looks Clean

File: authentication/forgot_password/forgot_password.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    justify-content: flex-start;
+}
+.form-components-container {
+  width: 50%;
+}
+  .password-eye-icon {
+    position: absolute;
+    top: 2.1rem;
+    right: 0.5rem;
+  }
```
Comment (Ankur9669) [inline authentication/forgot_password/forgot_password.css]: Please convert it into pixels

File: authentication/forgot_password/forgot_password.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    />
+    <link rel="stylesheet" href="/common.css" />
+    <link rel="stylesheet" href="/index.css" />
+    <link rel="stylesheet" href="/authentication/authentication_common.css" />
+    <link rel="stylesheet" href="forgot_password.css" />
+    <script
+      src="https://kit.fontawesome.com/482acc5102.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
```
Comment (Ankur9669) [inline authentication/forgot_password/forgot_password.html:20]: Could add script tag in the bottom

File: authentication/signin/signin.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  .password-eye-icon {
+    position: absolute;
+    top: 2.1rem;
+    right: 0.5rem;
+  }
+  .email-icon {
+    position: absolute;
+    top: 2.2rem;
+    right: 0.5rem;
+  }
```
Comment (Ankur9669) [inline authentication/signin/signin.css]: Please convert it into pixels

File: authentication/signin/signin.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    />
+    <link rel="stylesheet" href="/common.css" />
+    <link rel="stylesheet" href="/index.css" />
+    <link rel="stylesheet" href="/authentication/authentication_common.css" />
+    <link rel="stylesheet" href="signin.css" />
+    <script
+      src="https://kit.fontawesome.com/482acc5102.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
```
Comment (Ankur9669) [inline authentication/signin/signin.html:21]: Can use script tag in the bottom

File: authentication/signup/signup.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+.password-eye-icon{
+    position: absolute;
+    top:1.2rem;
+    right:0.5rem;
+}
+.email-icon{
+    position: absolute; 
+    top:2.5rem;
+    right:0.5rem;
+}
```
Comment (Ankur9669) [inline authentication/signup/signup.css]: Could use pixels here

**Review by Ankur9669 — APPROVED** [review]
The UI looks awesome, it is very clean and consise so you can go ahead and merge it

File: authentication/signin/signin.css (hunk 3 above)
Comment (Priya-Patel31) [reply authentication/signin/signin.css]: done !!

File: authentication/signup/signup.css (hunk 5 above)
Comment (Priya-Patel31) [reply authentication/signup/signup.css]: changed !!

File: authentication/forgot_password/forgot_password.css (hunk 1 above)
Comment (Priya-Patel31) [reply authentication/forgot_password/forgot_password.css]: converted !!

**Pull request merged**